crates/uv/tests/pip_install.rs

```diff
@@ -987,7 +986,7 @@ fn reinstall_build_system() -> Result<()> {
 
     ----- stderr -----
     Resolved 8 packages in [TIME]
-    Downloaded 7 packages in [TIME]
+    Downloaded 8 packages in [TIME]
```

Comment:
This is correct, because we no longer download `flit-core` during the resolution phase.